devchat.go

```diff
@@ -591,27 +592,32 @@ func runCommands(line string, u *user, isSlack bool) {
 		return
 	}
 	if strings.HasPrefix(line, "/hang") {
-		rest := strings.TrimSpace(strings.TrimPrefix(line, "/hang"))
-		if len(rest) > 1 {
-			u.writeln(u.name, line)
```

Comment:
see this line? it makes sure only the sending user sees their command line

Reply:
Yeah didn't notice.

Reply:
It's okay!